### ISSUE TITLE
Мат.склад - накладная на получение

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#1162]
Originally reported on Google Code with ID 1162

```
Неплохо было бы добавить в накладную расходы(транспортные и т.д.), как в 
торговле.


```

Reported by `www.sasha` on 2009-01-08 12:36:22

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2009-01-08 15:24:38
- **Labels added**: Type-Enhancement, Priority-Medium, Depot

**Comment by gsbelarus** [comment]
Reported by `danilchyk` on 2013-05-18 09:01:39
- **Status changed**: `Duplicate`
- **Merged into**: #1741
